core-bundle/assets/controllers/operations-menu-controller.js

```diff
@@ -71,39 +71,36 @@ export default class OperationsMenuController extends Controller {
         this.element.addEventListener('pointerup', (e) => e.stopPropagation(), { once: true });
 
         this.$menu.elements.submenuToggles[0].open();
-        this.setFixedPosition(event);
+        this.setPosition(event);
     }
 
-    setFixedPosition(event) {
-        const rect = this.submenuTarget.getBoundingClientRect();
-        let x;
-        let y;
-        let offset = 0;
-
-        if (event) {
-            x = event.clientX;
-            y = event.clientY;
-        } else {
-            const r = this.controllerTarget.getBoundingClientRect();
-            x = r.right;
-            y = r.bottom;
-            offset = 2;
-        }
+    setPosition(event) {
+        const offset = 2; // border-width that is excluded from getBoundingClientRect
 
-        this.submenuTarget.style.position = 'fixed';
-        this.submenuTarget.style.right = 'auto';
+        const submenuRect = this.submenuTarget.getBoundingClientRect();
+        const buttonRect = this.controllerTarget.getBoundingClientRect();
```

Comment:
Shouldn't we calculate the absolute position via the submenu's `offsetParent`? Using the button (its sibling) seems a bit arbitrary.

Reply:
Ideally it should already be `position: absolute` by its CSS (since it will now always be `position: absolute`, when visible). It being `position: fixed` is just a workaround for not having `overflow: hidden` on a root element, so that the overflow would not matter.

But that again could cause more issues down the line, so I guess we should just leave it like this.

Reply:
@fritzmg I actually tested using the `parentElement` but it would just change the following line due to the relative positioning on the button (which is already in place):

```diff
-    const buttonRect = this.controllerTarget.getBoundingClientRect();
+    const buttonRect = this.controllerTarget.parentElement.getBoundingClientRect();
```

I could rewrite it to using the row only for the `right click` event but would then need to reintroduce the `buttonRect` calc again for the button click for the correct offset. Summarized just the same amount of `getBoundingClient()` rewritten so it would work for both cases :/

Reply:
> @fritzmg I actually tested using the `parentElement`

Well it would need to be `offsetParent` in order to not rely on any element having non-static `position`.

Reply:
I see what you mean

Reply:
See b0244f11b0bdc4b22223fd2bff6e7892aa52021a